docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1380,7 +1380,9 @@ Affects the pruning of models that have not been updated for the given time
 duration. The value must be set to a multiple of the `bucket_span`. If set too
 low, important information may be removed from the model. Typically, set to
 `30d` or longer. If not set, model pruning only occurs if the model memory
-status reaches the soft limit or the hard limit.
+ status reaches the soft limit or the hard limit. For jobs created in 8.1 and
+ above, a default value of 30 days is set on job creation if no other value is
+ set.
```

Comment:
It looks like this is indented by 1 space, probably because of the way my earlier suggestion went in directly via GitHub.